src/main/CommandLine.cpp

```diff
@@ -1013,20 +1031,51 @@ int
 runWriteVerifiedCheckpointHashes(CommandLineArgs const& args)
 {
     std::string outputFile;
+    std::optional<std::string> trustedHashFile;
     uint32_t startLedger = 0;
     std::string startHash;
+    std::optional<std::uint32_t> fromLedger;
     CommandLine::ConfigOption configOption;
     return runWithHelp(
         args,
         {configurationParser(configOption), historyLedgerNumber(startLedger),
-         historyHashParser(startHash), outputFileParser(outputFile).required()},
+         historyHashParser(startHash), fromLedgerNumberParser(fromLedger),
+         trustedHashFileParser(trustedHashFile),
+         outputFileParser(outputFile).required()},
         [&] {
+            if (outputFile.empty())
+            {
+                LOG_ERROR(DEFAULT_LOG, "Must specify --output-file");
+                return 1;
+            }
+            if (fromLedger && trustedHashFile)
+            {
+                LOG_ERROR(DEFAULT_LOG, "Cannot specify both --from-ledger and "
+                                       "--trusted-hash-file");
+                return 1;
+            }
+            std::optional<LedgerNumHashPair> latestTrustedHashPair;
+            if (trustedHashFile)
+            {
+                if (!std::filesystem::exists(*trustedHashFile))
```

Comment:
This is redundant, as we check for file existence in `loadLatestHashPairFromJsonOutput`